documentation/messages/user.password_reset.transactional.md

```diff
@@ -0,0 +1,106 @@
+# User password reset transactional message
+
+User password reset event to trigger functionality within the Quicksilver system.
+
+----
+
+#### Current Configuration
+Accessed by connecting to RabbitMQ server and sending message in the following format.
+
+**Routing key**
+
+`user.password_reset.transactional`
+
+**Exchanges**
+- `transactionalExchange`
+
+**Consumers**
+
+| Consumer                                                                                 | Binding key           |
+| ---------------------------------------------------------------------------------------- | ----------------------|
+| [mbp-externalApplications-dashboard](../consumers/mbp-externalApplications-dashboard.md) |  `*.*.transactional`  |
+| [mbc-logging-gateway](../consumers/mbc-logging-gateway.md)                               |  `*.*.transactional`  |
+| [mbc-transactional-email](../consumers/mbc-transactional-email.md)                       |  `*.*.transactional`  |
```

Comment:
Confirmed:
- `mobileCommonsQueue`, `campaign.signup.*` and `user.registration.*` binding
- `userAPICampaignActivityQueue`, `campaign.*.*` binding
will not result in messages being sent with the key`user.password_reset.transactional`. These needed to be removed from `dosomething_mbp.install`: https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_mbp/dosomething_mbp.install#L167-L175